src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Square extends Figure {
+    public static final int MAXIMUM_SIZE = 50;
+    private int side;
+    private Random random = new Random();
+
+    @Override
+    public void randomizeProperty() {
+        side = random.nextInt(MAXIMUM_SIZE);
+    }
```

Comment:
The 'randomizeProperty' method should not exist. Instead, random values should be passed to the constructor of the figure. Remove this method and adjust the constructor accordingly.